packages/enzyme-adapter-react-13/src/ReactThirteenElementToTree.js

```diff
@@ -23,6 +24,8 @@ export default function elementToTree(el) {
   let rendered = null;
   if (Array.isArray(children)) {
     rendered = flatten(children, true).map(elementToTree);
+  } else if (isIterable(children) && typeof children !== 'string') {
+    rendered = flatten([...children], true).map(elementToTree);
```

Comment:
This is not currently working for nested iterables. E.g.

```
<div>
  { iterable1 }
  { iterable2 }
</div>
```

I think we might need a recursive flatten function that caters for either an array or an iterable:

```
export function flatten(arrs) {
  return arrs.reduce((flattened, item) =>
    flattened.concat(Array.isArray(item) || isIterable(item) ? flatten([...item]) : [item]), []);
}
```

Reply:
good call; recursion may be required here